itunes_rss.go

```diff
@@ -0,0 +1,196 @@
+package feeds
+
+// itunes rss support
+// validation done according to spec here:
+//    https://help.apple.com/itc/podcasts_connect/#/itcb54353390
+
+import (
+	"encoding/xml"
+	"fmt"
+	"time"
+)
+
+// private wrapper around the RssFeed which gives us the <rss>..</rss> xml
```

Comment:
All comments on exported types should start with typename - e.g “ITunesRSSFeedXML ...”